src/pages/contributing/get-started.mdx

```diff
@@ -0,0 +1,175 @@
+
+This work involves enhancing our components in code, like adding new features or
+variants. Designers are often responsible for researching UX and visual
+solutions and then creating a design spec for developers.
+[Here are issues](https://github.com/carbon-design-system/carbon/issues?q=is%3Aissue+is%3Aopen+label%3A%22proposal%3A+accepted%22+label%3A%22community+contribution%22+label%3A%22type%3A+enhancement+%F0%9F%92%A1%22+)
+in our enhancement backlog that need design or development work.
+
+### Design kits
+
+This work is all about creating and updating reusable components is Figma.
+You'll get valuable Figma experience and will collaborate with the Figma Guild.
+Check out the
+[status of design kits](https://www.figma.com/file/CFMtqV5Nztdbm0mi2UiDLg/Library-%2B-Template-Planning?type=design&node-id=3713-26762&mode=design&t=HqvAYXUeccKNMstT-0)
+and get involved by reaching out to the contacts listed.
+
+### Website documentation
+
+All components and patterns require usage, style, code, and accessibility
+guidance on the Carbon website. You can learn how to contribute documentation
+smoothly and utilize our markdown templates on the
+[documentation](contributing/documentation/) page. A list of components that
+still need documentation help can be found
+[here](https://github.com/carbon-design-system/carbon-website/issues/2085).
+
+### Fixing issues and bugs
+
+If you find a bug in the source code, you can help us by
+[filling out a bug report](https://github.com/carbon-design-system/carbon/issues/new/choose)
+and, ideally, submitting a pull request with a fix.
+
+### Icons and pictograms
+
+You can find instructions on how to create and contribute
+[UI icons](https://www.ibm.com/design/language/iconography/ui-icons/contribute),
+[app icons](https://www.ibm.com/design/language/iconography/app-icons/production),
+and
+[pictograms](https://www.ibm.com/design/language/iconography/pictograms/contribute)
+on the IBM Design Language site.
+
+## The process
+
+Whether you're contributing code, designs, or documentation, there's a process
+to follow. Here's what it looks like:
+
+### 1. Find a project to work on
+
+#### Volunteer for existing work
+
+The best way to volunteer is to look through
+[existing GitHub issues](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22)
+labeled with `community contribution`. You can also filter by
+[developer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+dev+%F0%9F%A4%96%22_)
+or
+[designer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+design+%3Apencil2%3A%22+)
+roles. Put a comment in the issue saying you'd like the help.
+
+The Carbon team is also happy to help you find a good first issue. Just stop by
+our [office hours](/whats-happening/meetups/) or send us a slack message.
```

Comment:
You could link to the anchor link for Office hours here if you want it to be more direct.